apis/core.oam.dev/v1alpha2/applicationcontext_types.go

```diff
@@ -28,7 +28,7 @@ type ApplicationContextSpec struct {
 
 // ApplicationContext is the Schema for the ApplicationContext API
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:shortName=appcontext,categories={oam}
+// +kubebuilder:resource:shortName=appcontext;context,categories={oam}
```

Comment:
the name `context` is common, I'm afraid it could conflict with other CRD resourcs.

Reply:
Done